### ISSUE TITLE
Run in Background

### ISSUE THREAD

**Issue by GoogleCodeExporter** [StonedRightMeow17/itransmission#10]
```
What steps will reproduce the problem?
1. Close program
2. Upon reopening, you will find torrent to have paused
3.

What is the expected output? What do you see instead?
The program should allow the torrent to continue in the background. At the very 
least allow for a toggle to turn this feature on or off.

What version of the product are you using? On what operating system?
0.5, iOS 4.3.3

Please provide any additional information below.


```

Original issue reported on code.google.com by `seansel...@gmail.com` on 6 Jun 2011 at 4:41
- Merged into: #6

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
One can get around this by editing "info.plist" in 
"/Applications/iTransmission". 
Once the file is open, change "UIApplicationExitsOnSuspend" to false. This will 
suspend it when in the background. Once this is done you can use Backgrounder 
to have your torrents run in the background. 
```

Original comment by `3radicat...@gmail.com` on 13 Jun 2011 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
Also, You will need to respring before these changes take effect.
```

Original comment by `3radicat...@gmail.com` on 13 Jun 2011 at 3:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `ccp0...@gmail.com` on 19 Jun 2011 at 1:44
- Changed state: **Duplicate**
